src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -795,7 +796,7 @@ private function getPropertyPath($propertyPath): PropertyPath
      *
      * @return AdapterInterface
      *
-     * @throws RuntimeException When the Cache Component isn't available
+     * @throws \RuntimeException When the Cache Component isn't available
```

Comment:
Looks like we are throwing a `LogicException`

Reply:
@fabpot, fixed